judge/views/ticket.py

```diff
@@ -236,6 +236,8 @@ def _get_queryset(self):
 
     def get_queryset(self):
         queryset = self._get_queryset()
+        if self.GET_with_session('hide'):
```

Comment:
`hide` isn't very descriptive in this context, perhaps `open`?

Reply:
Fixed. I think `open` is a better word.